doc/sphinxman/source/build_planning.rst

```diff
@@ -0,0 +1,1152 @@
+Unless using the conda package, which is relocatable, the installation
+directory must be specified with CMake variable ``CMAKE_INSTALL_PREFIX``
+before compiling.
+
+* Build with Specific Install Directory
+
+  .. code-block:: bash
+
+   cmake -DCMAKE_INSTALL_PREFIX=/nfs/common/software/psi4
+
+* Relevant CMake Options:
+
+  .. code-block:: bash
+
+   CMAKE_INSTALL_PREFIX=PATH  # Location to which Psi4 and internally built
+                              # add-ons are installed (default: /usr/local/psi4)
+
+
+.. ###<a name="profiling"></a>
+.. 
+.. How to set up a profiling build
+.. ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. 
+.. Specifying build type ``setup --type profile`` prepares a release build
+.. type with the addition of extra flags for linking against the profiler
+.. `gprof`.
+
+
+.. _`faq:setuptype`:
+
+How to compile for debugging
+----------------------------
+
+Flags to turn optimizations off and debugging on can be set across the
+project and plugins with CMake variable ``CMAKE_BUILD_TYPE`` before
+compiling. Note that these flags *will not* propagate to any add-ons that
+are detected pre-built rather than built.
+
+* Build without optimization
+
+  .. code-block:: bash
+
+    cmake -DCMAKE_BUILD_TYPE=debug
+
+                                  set the CMake build type [default: release]
+
+* Relevant CMake Options:
+
+  .. code-block:: bash
+
+    CMAKE_BUILD_TYPE=[debug|release]  # Build type (Release or Debug)" [default: release]
+
+
+
+.. .. _`faq:setupobjdir`:
+.. 
+.. How to compile elsewhere than ``{top-level-psi4-dir}/objdir``
+.. -------------------------------------------------------------
+.. 
+.. [How to choose the compilation directory, ``$objdir``](2_Planning#chooseobjdir)
+.. 
+.. * Build in Specific Directory
+.. 
+..   .. code-block:: bash
+.. 
+..    cd $top-level-psi4-dir
+..    cmake -H. -Bobj-gcc
+..    cd obj-gcc
+
+
+.. _`faq:erroreriam`:
+
+How to fix error "``RuntimeError: value for ERI``"
+--------------------------------------------------
+
+You will need to rebuild libint. Reissue ``cmake`` or edit
+``CMakeCache.txt`` with larger ``MAX_AM_ERI`` and rebuilt.
+
+* :ref:`faq:setupmaxameri`
+
+.. _`faq:chooseobjdir`:
+
+How to choose the compilation directory, ``{objdir}``
+-----------------------------------------------------
+
+* there is no default
+* common choices are ``objdir`` or ``build`` under :samp:`{top-level-psi4-dir}`
+
+  * ``cd {top-level-psi4-dir} && cmake -H. -Bobjdir``
+  * ``cd {top-level-psi4-dir} && cmake -H. -Bbuild``
+
+* in-source builds (``*.cc`` and ``*.o`` in same directory) are disallowed
+* builds *outside* :samp:`{top-level-psi4-dir}` are permitted
+
+
+.. _`faq:doconfigure`:
+
+How to save configuration settings for a future compilation
+-----------------------------------------------------------
+
+Create a file like ``do-configure`` with the ``cmake`` command and options
+*on one line*. ::
+
+ >>> cd {top-level-psi4-dir}
+ >>> cat do-configure
+     cmake -H. -Bobjdir5 \
```

Comment:
Good catch. Should be `{objdir}` when trying to be general and `objdir` when doing simplest prescribed path. Fixed the `objdir5`.